src/templates/plugin.jsx

```diff
@@ -60,10 +63,12 @@ function PluginPage({data: {jenkinsPlugin: plugin}}) {
                     </ul>
                     <div className="padded">
                         {state.selectedTab === 'documentation' && (<>
-                            <h1 className="title">
-                                {cleanTitle(plugin.title)}
+                            <h1 className="title-details">
```

Comment:
There should only be one h1 on the page, this section doesn’t appear to need to be a heading at all.

Can you add a screenshot of a plugin with active security warnings too?

Reply:
That would also help remove all this clutter from the heading in Google search results
https://www.google.com/search?q=jenkins+git+plugin&oq=jenkins+git+plugin

Reply:
Hi @timja, I tried to find some plugin with a security warning but didn't succeed. Can you help me out with some plugin names that have some security warning?

Reply:
@sachinmukherjee please check this one: https://plugins.jenkins.io/custom-job-icon/

Reply:
@zbynek I tried the custom job icon plugin but it is not showing any security warning in my development env.
![image](https://user-images.githubusercontent.com/19318218/93710285-06a48200-fb63-11ea-8eeb-0ed600a3e004.png)

Reply:
you may need to specify which plugin page to build when starting the dev server:
```
export GET_CONTENT_SINGLE='custom-job-icon';yarn dev
```
![image](https://user-images.githubusercontent.com/1105305/93714740-fab9bf80-fb64-11ea-9b10-8c51de005e13.png)

Reply:
Do I need to install some specific tool for this because when I ran `export GET_CONTENT_SINGLE='custom-job-icon';yarn dev` it gave me an error like `'export' is not recognized as an internal or external command,
operable program or batch file.`. I'm using windows 10

Reply:
export is a linux command
you can use set instead in windows

Reply:
After following the above steps security warning is still not coming up

Reply:
@timja the screenshot of a plugin with a warning is above -- do you think it's OK for the scope of this PR or should something be done here? Personally I think it's weird that current warnings are hidden (you need to click the triangle) and warnings for old versions are shown directly. It might make sense to kill the triangle entirely and show the current warning as an [alert](https://getbootstrap.com/docs/4.0/components/alerts/) above the tabs. Maybe out of scope here?

Reply:
the above looks ok I think, (in comparison to the previous)

yes it's very weird and needs a re-design

Reply:
Can I commit my files with the above mentioned changes?

Reply:
yes please